packages/wasmkit/src/lib/deploy/contract.ts

```diff
@@ -45,6 +45,25 @@ export class Contract {
   private checkpointData: Checkpoints;
   private readonly checkpointPath: string;
 
+  private loadInterval?: NodeJS.Timer;
+
+  private printLoadingAnimation (logMsg: string): void {
+    const loadingChars = ['/', '-', '\\', '|'];
```

Comment:
fix the typo of `\\`

Reply:
Sir I have used \\ as an escape sequence to print a single backslash character